new-lamassu-admin/src/pages/Transactions/Transactions.js

```diff
@@ -130,7 +130,7 @@ const Transactions = () => {
       textAlign: 'right',
       size: 'sm',
       view: it =>
-        `${toUnit(new BigNumber(it.cryptoAtoms), it.cryptoCode).toFormat(5)} ${
+        `${toUnit(new BigNumber(it.cryptoAtoms), it.cryptoCode)} ${
```

Comment:
We don't want to remove the rounding as a whole, we should do the same as the machine does. 

Also is there even enough space here to show the additional digits?